app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Ensure that the input string is split only once, as required by the checklist: 'Make sure to call each necessary method only once (for example, use `.split()` a single time).' This is already correctly implemented here.